README.md

```diff
@@ -17,14 +17,14 @@
 
 This is not a radio. It is a sample player that behaves like a radio.  
 
-A digital way to simulate radio-powered compositions by John Cage, Karlheinz Stockhausen and Don Buchla's voltage-controlled radio experiments. It uses a teensy3.1 to play files from a SD card to simulate a voltage controlled AM/FM/Shortwave/Time Travel radio. 
+A digital way to simulate the radio-powered compositions of John Cage and Karlheinz Stockhausen as well as Don Buchla's voltage-controlled radio experiments. It uses a [Teensy](https://www.pjrc.com/teensy/) to play files from an SD card to simulate a voltage controlled AM/FM/Shortwave/Time Travel radio. 
```

Comment:
Open to discuss the wording here :) However, the Teensy version I received is 3.2 rather than 3.1, so at least something should be done about this.